libopenarc/arc.c

```diff
@@ -834,6 +834,11 @@ arc_init(void)
 void
 arc_close(ARC_LIB *lib)
 {
+	if (lib->arcl_signre)
+	{
+		(void) regfree(&lib->arcl_hdrre);
+		lib->arcl_signre = FALSE;
```

Comment:
Given that `lib` is immediately freed, the main purpose of this assignment appears to be confusing people who read the code. I recommend either adding a comment explaining that it doesn't do anything useful, or dropping the line entirely.

Reply:
There are additional resources to free as it turns out, shifting to using public API to free them.